src/libraries/System.Formats.Asn1/src/System.Formats.Asn1.csproj

```diff
@@ -2,7 +2,8 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
+    <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
```

Comment:
if we're excluding NetCoreAppCurrent from the package how is this going to fix the issue?

Reply:
The assembly is listed in https://github.com/dotnet/runtime/blob/master/src/libraries/NetCoreAppLibrary.props; and my understanding is that the shared runtime will get the $(NetCoreAppCurrent) build instead of the netstandard2.0 build, which means it will reference System.Runtime and friends instead of netstandard.dll, and therefore netstandard.dll won't be part of its own dependency chain.

I very well could be wrong about any or all of this :smile:.

Reply:
Makes sense. I didn't know it was listed as an inbox assembly.